tests/wpt/web-platform-tests/dom/nodes/NodeList-Iterable.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>NodeList Iterable Test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+    <p>wake up</p>
+<script>
+    var paragraphs;
+    setup(function() {
+        paragraphs = document.querySelectorAll('p');
```

Comment:
However, `getElementsByName` is implemented, so instead of depending on `querySelectorsAll`, you may want to use the former instead.